hardhat.config.ts

```diff
@@ -54,6 +59,10 @@ const config: HardhatUserConfig = {
     outDir: 'types',
     target: 'ethers-v5',
   },
+  dependencyCompiler: {
+    paths:['@pooltogether/pooltogether-proxy-factory/contracts/GenericProxyFactory.sol'],
+    keep: true
```

Comment:
I don't think keep is needed...doesn't this just leave artifacts in the contracts/ directory?